daisy/compute/compute.go

```diff
@@ -154,9 +158,10 @@ type clientImpl interface {
 }
 
 type client struct {
-	i   clientImpl
-	hc  *http.Client
-	raw *compute.Service
+	i       clientImpl
```

Comment:
No need to change anything now but we might want to avoid aligning the variable types since it means adding a new variable shows a change for all the others that have to be realigned.

Reply:
This is gofmt, we can't really not do it (unless we add spaces between the fields, which is odd). This is actually more of an issue with the diff in github, these shouldn't show up as new lines but instead just changed lines